### PR TITLE
Feature/storage/stg73 regenerate swagger for blobs

### PR DESCRIPTION
**Pull request by kasobol-msft**
- regenerate swagger from latest spec
- fix blob swagger Readme.md

### CODE REVIEW

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ListContainersIncludeType.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob.implementation.models;
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ListContainersIncludeType.java:5]: Looks like this is double generated? Shouldn't there only be the `ListBlobContainersIncludeType` in the public API and this isn't needed?

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/ListContainersIncludeTypeWrapper.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob.implementation;
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/ListContainersIncludeTypeWrapper.java:5]: Should this and `ListBlobContainersIncludeTypeWrapper` both exist? Should this be removed and the XML node names be fixed in the `BlobContainers` variant?

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ListContainersIncludeType.java (hunk 1 above)
Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ListContainersIncludeType.java]: Removed.

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/ListContainersIncludeTypeWrapper.java (hunk 2 above)
Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/ListContainersIncludeTypeWrapper.java]: Good point. Ideally both are not needed.
However, I didn't find a way to force generator to skip `ListBlobContainersIncludeTypeWrapper`, so I end result has just one wrapper.

I can see that similar issue exist for existing `ListBlobsIncludeItemWrapper` and `ListBlobsIncludeItem` pair - the wrapper is not used but exist.

**Pull request merged**